dev/com.ibm.ws.security.acme_fat/publish/files/test-ca2.pem

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Probably rename this file. What is this file for?

Reply:
That's what boulder uses to get the intermediate certificate.